install_cfgov_refresh.sh

```diff
@@ -2,6 +2,7 @@
 
 git clone --depth 1 https://github.com/cfpb/cfgov-refresh
 cd cfgov-refresh
+python -m pip install --upgrade pip
```

Comment:
I don't think this is necessary — IMHO, it should be up to the environment that `tox` is running in to provide an up-to-date `pip`.

Reply:
I agree. Which file defines the version of pip to install.

Reply:
That's up to the local environment that `tox` is installed in. I don't think we need to be in the business of managing that for developers, Travis, Jenkins, etc here.